src/main/java/client/processor/npc/DueyProcessor.java

```diff
@@ -285,7 +285,7 @@ public static void dueySendItem(Client c, byte invTypeId, short itemPos, short a
         if (c.tryacquireClient()) {
             try {
                 int fee = Trade.getFee(sendMesos);
-                if (sendMessage.length() > 100) {
+                if (sendMessage != null && sendMessage.length() > 100) {
```

Comment:
Is this a good idea?

`if (quick && sendMessage.length() > 100) {`

Reply:
It works, but it's more brittle.